src/applications/facility-locator/tests/e2e/mobile.cypress.spec.js

```diff
@@ -6,10 +6,18 @@ Cypress.Commands.add('checkSearch', () => {
   cy.axeCheck();
 
   // Search
+  cy.get('#street-city-state-zip', { timeout: 10000 })
+    .should('exist')
+    .should('not.be.disabled')
+    .clear({ waitForAnimations: true, force: true });
```

Comment:
Again, I think `force: true` overrides `waitForAnimations: true`, no?

Reply:
It's hard to say from the documentation, but it might override it. The `waitForAnimations: true` is not necessary with `force: true`; I was experimenting with both to see which one fixes the failure. I've pushed up a change to make this consistent. I ran the test 50 times (in the browser and headless) without any failures.